source/services/dataContracts/converters/aliasConverter/aliasConveter.tests.ts

```diff
@@ -0,0 +1,20 @@
+import { AliasConverter } from './aliasConverter';
+
+describe('aliasConverter', (): void => {
+	let aliasConverter: AliasConverter<string>;
+
+	beforeEach((): void => {
+		aliasConverter = new AliasConverter<string>('propValue');
+	});
+
+	it('should get the property from the parent context using the specified property name', (): void => {
+		let parent: any = { propValue: 'value' };
+		expect(aliasConverter.fromServer(undefined, parent)).to.equal('value');
+	});
+
+	it('should set the value on the parent context using the specified property name', (): void => {
+		let parent: any = {};
+		aliasConverter.toServer('value', parent);
```

Comment:
So you could have it so if an object is returned, the converterService assumes that it's a clone of the parent, but then you would have two issues:
1) You would now be restricted to only primitive converters. Currently it's flexible enough that you can use a custom converter at any point in the tree to map a whole child object.
2) How would you handle it if some of the converters have already run, but not all of them? For example in the following case:
```
let transform: any = {
    lastUpdated: __converters.dateConverter,
    value: new __converters.AliasConverter('valueFromServer'),
    created: __converters.dateConverter,
};
```
The last updated converter would run, but then once the alias converter runs the service would have to decide if it needs to rerun the converters or just leave the data as-is. Unless you somehow tracked which converters have already completed.